display/ssd1306.h

```diff
@@ -330,47 +343,48 @@ class StandardDisplayController : public DisplayControllerBase<Width, col_t>, Sa
       default:
       case SCREEN_UNSET:
       case SCREEN_OFF:
-	Clear();
+        Clear();
         return 3600000; // Long time!
 
       case SCREEN_STARTUP:
-	Clear();
+        Clear();
         // DrawText("==SabeR===", 0,15, Starjedi10pt7bGlyphs);
         // DrawText("++Teensy++",-4,31, Starjedi10pt7bGlyphs);
-	if (WIDTH < 128) {
-	  display_->DrawText("p-os", 0,15, Starjedi10pt7bGlyphs);
-	} else {
-	  display_->DrawText("proffieos", 0,15, Starjedi10pt7bGlyphs);
-	}
-	display_->DrawText(version,0,31, Starjedi10pt7bGlyphs);
-	if (HEIGHT > 32) {
-	  display_->DrawText("installed: ",0,47, Starjedi10pt7bGlyphs);
-	  display_->DrawText(install_time,0,63, Starjedi10pt7bGlyphs);
-	}
+        if (WIDTH < 128) {
+          display_->DrawText("p-os", 0,15, Starjedi10pt7bGlyphs);
+        } else {
+          display_->DrawText("proffieos", 0,15, Starjedi10pt7bGlyphs);
+        }
+        display_->DrawText(version,0,31, Starjedi10pt7bGlyphs);
+        if (HEIGHT > 32) {
+          display_->DrawText("installed: ",0,47, Starjedi10pt7bGlyphs);
+          display_->DrawText(install_time,0,63, Starjedi10pt7bGlyphs);
+        }
         next_screen_ = SCREEN_PLI;
-        return font_config.ProffieOSFontImageDuration;
+        return font_config.ProffieOSBootImageDuration;
```

Comment:
This needs to be backwards compatible, so if the "ProffieOSBootImageDuration" isn't set, then it should still use ProffieOSFontImageDuration.

Reply:
But if we have a default Boot duration set in hybrid_font, wouldn't that take care of the case where BootImageDuration isn't set in the font config.ini?
Well regardless, in testing all this, setting this to a custom duration here actually seems pretty pointless, as the screen doesn't even initialize fast enough for most wav sounds anyway, and we should probably assure there's enough time to read the info....
So how about we just make it so that if the startup splash screen is going to show because there's no boot image, we just show it for like 4 seconds, period?

Reply:
addressed below.